packages/forms/src/widgets/DatalistWidget/DatalistWidget.js

```diff
@@ -168,7 +194,27 @@ class DatalistWidget extends React.Component {
 	}
 
 	initSuggestions(value) {
-		const suggestions = getMatchingSuggestions(this.props.schema.enum, value);
+		let items;
+		if (this.props.schema.enum) {
+			items = this.props.schema.enum;
+		} else if (this.props.formContext && this.props.formContext.fetchItems) {
+			items = this.props.formContext.fetchItems(this.props.schema.title);
```

Comment:
Something based on the title :(
But right now we don't have anything else, so I understand. That may change with next version of forms.